src/build/spawn_stream.mjs

```diff
@@ -23,7 +23,7 @@ export const spawnStream = (command, ...parameters) =>
     const stdout = [];
     const stderr = [];
 
-    console.debug(`Running [${[command, ...parameters.map(e => `'${e}'`)].join(' ')}]`);
+    console.debug(chalk.gray(`Running [${[command, ...parameters.map(e => `'${e}'`)].join(' ')}]...`));
```

Comment:
*nit* will this print secrets to the debug output? I guess if we are using environment variables then it's ok.